src/pcapi/notifications/push/transactional_notifications.py

```diff
@@ -40,3 +42,19 @@ def get_bookings_cancellation_notification_data(booking_ids: List[int]) -> Optio
             body=f"""Ta {cancelled_object} "{bookings[0].stock.offer.name}" a été annulée par l'offreur.""",
         ),
     )
+
+
+def get_tomorrow_booking_notification_data(booking_id: int) -> Optional[TransactionalNotificationData]:
+    booking = Booking.query.filter_by(id=booking_id).join(Booking.stock).join(Stock.offer).first()
+
+    if not booking or not booking.stock or not booking.stock.offer:
```

Comment:
Un booking a forcément un stock et un stock a forcément une offre. Donc on pourrait seulement écrire `if not booking`.

Mais quel est le cas où on arrive ici avec un identifiant qui n'existe pas ? (Pour info, on ne supprime pas de données, ou alors très exceptionnellement. Il est dont très (très) improbable qu'une réservation ait été supprimée entre le lancement du cron et l'exécution du job par le worker.)

Par contre, avec le délai du job, on pourrait se retrouver avec une réservation que l'utilisateur a annulé entre temps. Du coup, suggestion :

```suggestion
    if booking.isCancelled:
        return None
```